modules/akka-http/src/main/scala/sup/modules/akkaHttp.scala

```diff
@@ -47,13 +46,13 @@ object akkaHttp {
   def healthCheckRoutesWithContext[F[_]: Functor, H[_] : Reducible, R](
     healthCheck: HealthCheck[F, H],
     path: String = "health-check",
-    f: RequestContext => F ~> Future)
-  (implicit marshaller: ToEntityMarshaller[HealthResult[H]], combineHealthChecks: Semigroup[Health]): Route = {
+    run: RequestContext => F ~> Future)
```

Comment:
I think this might infer better if it's in a separate parameter list (F will be inferred after the first parameter list).

Reply:
I'm not sure if it makes a difference. `F` already appears in the `healthCheck` parameter. But at least like this the syntax is somewhat nicer on the call-site.

Reply:
I'm not convinced...

```scala
@ def foo[A](a: A, f: A => String): String = f(a)
defined function foo

@ foo(5, _ => "foo")
cmd1.sc:1: missing parameter type
val res1 = foo(5, _ => "foo")
                  ^
Compilation Failed
```

but as it's a lambda it's probably best to keep it separate anyway :P